### PR TITLE
chore: update example apps with new `getPages` method

### PR DESCRIPTION
**Pull request by arvinpoddar**
Currently using the snapshot version of the runtime to test everything. Will modify this PR when the new version of the runtime is released.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (migueloller) [inline pnpm-lock.yaml:1]: How come the root `pnpm-lock.yaml` changed? None of the examples are pnpm workspaces so I wouldn't expect a change here at all.

**Review by migueloller — APPROVED** [review]
Looks good! Just had a couple of questions. Also, can we verify that the Vercel deployments for all these examples deployed without issues?

**Comment by arvinpoddar** [comment]
> Looks good! Just had a couple of questions. Also, can we verify that the Vercel deployments for all these examples deployed without issues?

Took a look at the preview deployments and the build logs, everything is looking good. I also cross-referenced some of these deployments with some from previous PRs, and the number of statically generated pages is the same. Feeling good!

File: examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx
[the remark this answers is not in the document]
Comment (arvinpoddar) [reply examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx:1]: The catalyst example has its own Prettier config that is different from the others. Their config enforces semicolons, while the others don't. I think this file managed to get by somehow - perhaps it was previously saved without formatting.

File: pnpm-lock.yaml
Comment (arvinpoddar) [reply pnpm-lock.yaml:1]: Yep, that was my mistake. Thanks for catching that.

File: examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+    .toArray();
 }
 
 export default async function Page({ params }: { params: ParsedUrlQuery }) {
-  const path = '/' + (params?.path ?? []).join('/')
+  const path = '/' + (params?.path ?? []).join('/');
   const snapshot = await client.getPageSnapshot(path, {
     siteVersion: getSiteVersion(),
-  })
+  });
```
Comment (agurtovoy) [inline examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx:23]: Semicolons ^^

File: examples/bigcommerce-catalyst/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "@bigcommerce/catalyst-client": "^0.1.1",
     "@graphql-typed-document-node/core": "^3.2.0",
     "@icons-pack/react-simple-icons": "^9.3.0",
-    "@makeswift/runtime": "^0.18.0",
+    "@makeswift/runtime": "^0.19.0",
```
Comment (agurtovoy) [inline examples/bigcommerce-catalyst/package.json:20]: The gotcha with the `bigcommerce-catalyst` example is that it's the only example where we store `package-lock.json` in the repo, so we need to update that as well.

**Review by agurtovoy — APPROVED** [review]
Looking good, just a couple of comments

File: examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx (hunk 1 above)
Comment (arvinpoddar) [reply examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx:23]: The catalyst example has a separate prettier config, so the auto-formatting is part of the change - the rest of the files in this example have semicolons. I believe this file somehow made it past auto-formatting.

File: examples/bigcommerce-catalyst/package.json (hunk 2 above)
Comment (arvinpoddar) [reply examples/bigcommerce-catalyst/package.json:20]: Good catch, thank you. I should have been using `npm` when upgrading the runtime instead of `pnpm`

**Review by agurtovoy — APPROVED** [review]

**Pull request merged**